span-normalizer/helm/values.yaml

```diff
@@ -115,11 +132,12 @@ kafka-topic-creator:
   kafka:
     topics:
       - name: raw-spans-from-jaeger-spans
-        replicationFactor: 2
+        replicationFactor: 3
```

Comment:
I am interested on this change 2->3. Any particular reason?

Reply:
Recommended and default replication factor itself is 3.